packages/gatsby-transformer-screenshot/README.md

```diff
@@ -83,3 +83,19 @@ First, you will need to [create a S3 bucket](https://docs.aws.amazon.com/AmazonS
 To build the Lambda package, run `npm run build-lambda-package` in this directory. A file called `lambda-package.zip` will be generated - upload this as the source of your AWS Lambda. Finally, you will need to set `S3_BUCKET` as an environment variable for the lambda.
 
 To set up the HTTP interface, you will need to use AWS API Gateway. Create a new API, create a new resource under `/`, select "Configure as proxy resource", and leave all the settings with their defaults. Create a method on the new resource, selecting "Lambda Function Proxy" as the integration type, and fill in the details of your lambda.
+
+## Placeholder image
+
+If your site pulls a lot of screenshots it might be beneficial to use placeholder image instead of downloading and processing all the screenshots. It will help with data sourcing and query running times.
+
+You can use placeholder image by setting `GATSBY_SCREENSHOT_PLACEHOLDER` environment variable when running `gatsby develop`:
+
+```shell
+GATSBY_SCREENSHOT_PLACEHOLDER=1 gatsby develop
+```
+
+or by adding it to `.env.development` file in root of your project:
+
+```shell:title=.env.development
```

Comment:
This title syntax isn't supported here :( Wish it was everywhere, haha.

Reply:
README for packages is shown in plugin library on gatsbyjs.org:
![screen shot 2018-10-15 at 22 17 16](https://user-images.githubusercontent.com/419821/46975911-2655e480-d0c8-11e8-9003-a9cc6e3cd755.png)